### PR TITLE
add dualtor flag for pfcwd cases

### PR DESCRIPTION
**Pull request by lipxu**
### Description of PR

Summary:
Fixes # (issue)
PFCWD cases on dualtor failed

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [x] 202205

#### What is the motivation for this PR?
PFCWD cases on dualtor failed

#### How did you do it?
resolve_arp function needs to call the CMD "docker exec -i swss arping ...", but it would failed on standby dualtor

#### How did you verify/test it?
Run original case, no such failure

#### Any platform specific information?
Dualtor testbeds.

### CODE REVIEW

File: tests/pfcwd/test_pfcwd_function.py [hunk 1]
```diff
@@ -277,7 +277,7 @@ def verify_pkt_cnts(self, port_type, pkt_cnt):
 
 class SetupPfcwdFunc(object):
     """ Test setup per port """
-    def setup_test_params(self, port, vlan, init=False, mmu_params=False):
+    def setup_test_params(self, port, vlan, init=False, mmu_params=False, is_dualtor=False):
```
Comment (XuChen-MSFT) [inline tests/pfcwd/test_pfcwd_function.py:280]: can check if it is dual tor in tbinfo instead of change methor definition

Comment (lipxu) [reply tests/pfcwd/test_pfcwd_function.py]: Sure, update the code.

**Review by XuChen-MSFT — APPROVED** [review]

**Comment by wangxin** [comment]
@lipxu Cherry-picking this PR to 202012 branch got too many merge conflicts that are hard to resolve. Can you create a separate PR to the 202012 branch?

**Comment by lipxu** [comment]
> @lipxu Cherry-picking this PR to 202012 branch got too many merge conflicts that are hard to resolve. Can you create a separate PR to the 202012 branch?

Sure, will submit a separate PR to 202012 branch later, thanks.

**Pull request merged**